README.md

```diff
@@ -1,2 +1,54 @@
 # transifex-native-sandbox
 A collection of sample applications integrated with transifex-native
+
+# What is Transifex Native?
+
+A new way to localise software within the code - no files no deployments. A [video](https://www.youtube.com/watch?v=SzEOCKNjRZU) is worth a thousand words!
+But if you are a words person here is the [documentation](https://docs.transifex.com/transifex-native-sdk-overview/introduction).
+
+# Try it out yourself
+
+  
+  - [x] Step 1: Clone the repo of your preference (Python, django, JS, React or React Native)
+  
+
+  - [x] Step 2: Set things up using this guide: it's literally 3 commands. 
+  <ADD GUIDE here>
+
+  If you are not up to it we've built a script to do it for you. Just do <command>
+  
+
+  - [x] Step 3: Push your strings to Transifex with this <command>
+
+  Note :We've enable MT so that you don't have to do the translations yourself.
+  
+  
+  - [x] Step 4: Pull your translations.
```

Comment:
I thought the point of transifex native was that you don't have to pull the translations :slightly_smiling_face: